demos/lineal_spring.py

```diff
@@ -44,16 +44,15 @@
 #
 # We begin by importing from the namespaces of Firedrake and Movement. ::
 
-from firedrake import *
-from movement import *
```

Comment:
The demo no longer flows correctly

Reply:
Sorry @jwallwork23, I don't see what you mean by "flows" :) could you please clarify?

Reply:
The demo says "We begin by importing ..." but the import statements no longer appear directly after that comment. So if you're reading the demo then it won't make sense.

Reply:
Ah sorry, it didn't occur to me to check for that! Fixed in cc701a6a772377897bbfb9ee6e709424d8356d8a . I cancelled the workflow since I just reordered the comment block.

Reply:
No worries, I checked the other demos in the other PRs and they were all fine - just this one.

Reply:
Changes look good but what happened to the CI? Did you cancel it?

Reply:
Oops yeah sorry missed that. Thanks!